README.md

```diff
@@ -1,6 +1,7 @@
 # scratch-resources
 Source files and images for translators to translate Scratch materials, published
```

Comment:
The content of the read me is getting kind of out of date.
Some updates could be helpful. E.g. "scratch-resources" header content replacement:
* resources/source: archive of source files for Scratch resources, including the Card Decks, Educator Guides and translations sent to us by translators.
* resources/www: published (PDF) resources on www. All resources are organized by locale - Originals are in the `/en` directory. We only publish PDFs for maximum portability.